perception/navigator_perception/nodes/camera_to_lidar/CameraLidarTransformer.cpp

```diff
@@ -44,7 +44,7 @@ bool CameraLidarTransformer::transformServiceCallback(
     return true;
   }
   geometry_msgs::TransformStamped transform =
-      tfBuffer.lookupTransform("right_right_cam", "velodyne", req.header.stamp);
+      tfBuffer.lookupTransform(req.header.frame_id, "velodyne", req.header.stamp);
```

Comment:
pcl_ros has a [transformPointCloud function](http://docs.ros.org/jade/api/pcl_ros/html/namespacepcl__ros.html#aad20af010640a41f1898e75f6a6121ad) that takes in a tf_listener and would clean this up nicely.